src/Heading/Heading.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Heading.scss';
+
+class Heading extends React.Component {
```

Comment:
It's good practice to make all your components functional (ie not class) components when you create them and only make them class components when you need the features of a class component (ie state, lifecycle methods etc). So i would make this a functional component for now